packages/@okta/vuepress-site/use_cases/inline_hooks/token_hook/token_hook/index.md

```diff
@@ -25,21 +25,21 @@ For steps to enable this inline hook, see below, [Enabling a Token Inline Hook](
 
 ## About
 
-This type of inline hook is triggered when OAuth 2.0 and OpenID Connect tokens are minted by your Okta Custom Authorization Server. Before sending the token to the requester, Okta calls out to your external service, and your service can respond with commands to add custom claims to the token.
+This type of inline hook is triggered when OAuth 2.0 and OpenID Connect (OIDC) tokens are minted by your Okta Custom Authorization Server. Before sending the token to the requester, Okta calls out to your external service, and your service can respond with commands to add custom claims to the token.
 
 This functionality can be used to add data that is sensitive, calculated at runtime, or complexly-structured and not appropriate for storing in Okta user profiles. Data added this way is never logged or stored by Okta. As an example, tokens minted for a medical app could be augmented with confidential patient data provided by your external service and not stored in Okta.
 
-This inline hook works only when using an Okta Custom Authorization Server, not the built-in Okta Authorization Server.
+In addition to adding custom claims, you can modify or remove an existing custom claim or an OIDC standard profile claim. You can also update how long an access token or an ID token is valid.
 
-You cannot use this inline hook to overwrite claims in tokens, only to add new ones.
+This Inline Hook works only when using an Okta Custom Authorization Server, not the built-in Okta Authorization Server.
```

Comment:
can we include link/reference to Okta Custom AS description or how to configure?